lang/en/texts/contribute-to-open-food-facts-by-playing.html

```diff
@@ -0,0 +1,73 @@
+<!-- no side column -->
+
+<h1>Play and contribute to Open Food Facts!</h1>
+
+<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteer developers.</h2>
+
+<div class="button_div unselectbuttondiv_front_en">
+	<a href="https://hunger.openfoodfacts.org/" class="button">Play now</button></a>
+<div>
+<p>We thank the many volunteers who have already <a href="https://world-en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
+<p><strong>This game allows you to modify products easily, quickly and in a fun way</strong>. Enough to make contributing to an Open Source project addictive.</p>
+<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions of people to choose better foods as well as public health research teams for their studies.</p>
```

Comment:
the "better foods as well as public health research teams" is hard to follow, I think it will be better with an explicit comma to separate the 2 parts:

```suggestion
<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions of people to choose better foods, and by public health research teams for their studies.</p>
```